homeassistant/components/plant.py

```diff
@@ -237,11 +293,59 @@ def state_attributes(self):
         sensor in the attributes of the device.
         """
         attrib = {
-            ATTR_ICON: self._icon,
             ATTR_PROBLEM: self._problems,
+            ATTR_SENSORS: self._readingmap,
+            CONF_UNIT_OF_MEASUREMENT: self._unit_of_measurement,
```

Comment:
As Andrey set, this is not allowed. you cannot use CONF_UNIT_OF_MEASUREMENT or ATTR_UNIT_OF_MEASUREMENT. Name it something else.

Reply:
ACK